src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -42,6 +42,7 @@
 _MANAGED_BY_TENANTS = 'managedByTenants'
 _USER_ENTITY = 'user'
 _USER_NAME = 'name'
+_HOME_ACCOUNT_ID = 'home_account_id'
```

Comment:
Let's use `homeAccountId` to sync with the rest of the code. 
```suggestion
_HOME_ACCOUNT_ID = 'homeAccountId'
```